### PR TITLE
Scopy for armhf

### PR DESCRIPTION
**Pull request by bindea-cristian**
Added: 
 - cross compile toolchain
 - sysroot configuration script
 - Qt cross compile script
 - cross compile Scopy dependencies
 - docker image building script
 - github workflow that cross compiles Scopy and generates an Scopy.AppImage artifact

### CODE REVIEW

File: ci/kuiper/scopy.desktop [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
```
Comment (adisuciu) [inline ci/kuiper/scopy.desktop:2]: V2.0

File: ci/kuiper/create_sysroot.sh [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+	sudo rm -rf /mnt/kuiper
+	rm -rf ${STAGING_AREA:?}/${IMAGE_FILE}
+	rm -rf ${STAGING_AREA}/image*.zip
+}
+
+
+# execute chroot inside the sysroot folder and install/remove packages using apt
+configure_sysroot(){
+cat << EOF | sudo chroot ${SYSROOT}
+export DEBIAN_FRONTEND=noninteractive
```
Comment (adisuciu) [inline ci/kuiper/create_sysroot.sh:47]: Move this to a file ?

File: ci/kuiper/build_scopy_kuiper.sh [hunk 3]
```diff
@@ -65,24 +65,28 @@ build_with_cmake() {
...
+	[ -d 'gr-m2k' ]		|| git clone --recursive https://github.com/analogdevicesinc/gr-m2k.git -b $GRM2K_BRANCH gr-m2k
+	[ -d 'qwt' ]		|| git clone --recursive https://github.com/cseci/qwt.git -b $QWT_BRANCH qwt
+	[ -d 'libsigrokdecode' ] || git clone --recursive https://github.com/sigrokproject/libsigrokdecode.git -b $LIBSIGROKDECODE_BRANCH libsigrokdecode
+	[ -d 'libtinyiiod' ]	|| git clone --recursive https://github.com/analogdevicesinc/libtinyiiod.git -b $LIBTINYIIOD_BRANCH libtinyiiod
+	[ -d 'iio-emu' ]	|| git clone --recursive https://github.com/analogdevicesinc/iio-emu -b $IIOEMU_BRANCH iio-emu
+	[ -d 'scopy' ]		|| git clone --recursive https://github.com/analogdevicesinc/scopy -b $SCOPY_BRANCH scopy
 	popd
 }
 
 instal_apt() {
```
Comment (adisuciu) [inline ci/kuiper/build_scopy_kuiper.sh:83]: typo

**Review by adisuciu — CHANGES_REQUESTED** [review]

File: ci/kuiper/build_scopy_kuiper.sh [hunk 4]
```diff
@@ -8,17 +8,17 @@ JOBS=-j1
...
 VOLK_BRANCH=main
 GNURADIO_BRANCH=maint-3.10
 GRSCOPY_BRANCH=3.10
 GRM2K_BRANCH=master
 LIBSIGROKDECODE_BRANCH=master
-QWT_BRANCH=qwt-multiaxes
+QWT_BRANCH=qwt-multiaxes-updated
 LIBTINYIIOD_BRANCH=master
-
+IIOEMU_BRANCH=master
```
Comment (AlexandraTrifan) [inline ci/kuiper/build_scopy_kuiper.sh:21]: could we change the reference to "main"? (i updated the default branch on iio-emu to be main now)
we should do this for all that apply

**Review by adisuciu — APPROVED** [review]

**Review by AlexandraTrifan — APPROVED** [review]

**Comment by adisuciu** [comment]
@Adrian-Stanea  can you test this ?

**Comment by Adrian-Stanea** [comment]
Tested [this AppImage](https://github.com/analogdevicesinc/scopy/actions/runs/8799696718/artifacts/1439168392) and no issues were found.

**Review by Adrian-Stanea — APPROVED** [review]
Tested [this AppImage](https://github.com/analogdevicesinc/scopy/actions/runs/8799696718/artifacts/1439168392) and no issues were found.

**Pull request merged**